midi_app_controller/config_files/controllers/x_touch_mini.yaml

```diff
@@ -1,9 +1,12 @@
 name: X-TOUCH MINI
+description: None
 button_value_off: 0
 button_value_on: 127
 knob_value_min: 0
 knob_value_max: 127
 default_channel: 11
+preferred_midi_in: None
+preferred_midi_out: None
```

Comment:
These fields are equal to the **string** `None` now. If we want them to be `None` in Python, they should be deleted from here and `= None`  should be added in the declaration of these attributes.